hashing/src/lib.rs

```diff
@@ -67,6 +94,12 @@ impl Digest {
         Digest(ret)
     }
 
+    // Temporarily unused, see comments inside `Digest::hash()` for details.
+    #[allow(unused)]
```

Comment:
```suggestion
    #[allow(unused)]
    #[inline(always)]
```

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/2119/commits/30511b6db38e134d19a399bb9c0b9058d31f3903